lib/gui/pages/finish/directives/banner-view.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+const hideWebview = (elem) => {
+
+  // We hide with width and height instead of display as that may be buggy
+  // with <webview>s.
```

Comment:
Oh yeah I should've linked this: https://electron.atom.io/docs/api/webview-tag/#css-styling-notes